.github/workflows/client.yml

```diff
@@ -27,6 +29,6 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: '10.x'
-      - run: npm install -g yarn
-      - run: yarn --cwd client/reactnative-proto
-      - run: yarn --cwd client/reactnative-proto test
+      - run: cd client/ionic && npm install
```

Comment:
are we confirming `npm` over `yarn`? happy to do so as I don't have a strong preference, just want to make sure

Reply:
yes, i noticed that as well.

Reply:
Actually I prefer Yarn but the checked in ionic system uses npm and not yarn, so I'm matching the code as written.  If we want to change the system to use yarn we should do that separately.  The CI should match the code either way.  SG?